site/gdocs/ArticleBlock.tsx

```diff
@@ -158,6 +156,17 @@ export default function ArticleBlock({
             />
         )
     }
+    if (block.type === "image" && block.dataErrors.length > 0) {
```

Comment:
Eventually we'll probably want a more robust abstraction for these errors but I didn't want to prematurely optimise